### DISCUSSION TITLE
Integrating with other packages

### DISCUSSION THREAD

**Discussion by Kokil** [spatie/laravel-multitenancy#271] [Q&A] [answered]
Hi, 

Actually, this is a question instead of an issue. I don't find any issue or article what is the issue that we are going to face while Integrating with other packages. I am going to make a multi-tenancy application, are we going to face package issues. for example, I am going to user spatie permission package I might need to use that to both tenant and landlord. So what is your answer in this context.

**Comment by masterix21** [comment, marked as answer]
Hi @Kokil, check here: https://github.com/spatie/laravel-multitenancy/issues/222
